larray/core/group.py

```diff
@@ -399,18 +399,18 @@ def _to_ticks(s, parse_single_int=False):
 
     Examples
     --------
-    >>> _to_ticks('M , F')                      # doctest: +NORMALIZE_WHITESPACE
-    array(['M', 'F'], dtype='<U1')
-    >>> _to_ticks('A,C..E,F..G,Z')              # doctest: +NORMALIZE_WHITESPACE
-    array(['A', 'C', 'D', 'E', 'F', 'G', 'Z'], dtype='<U1')
-    >>> _to_ticks('U')                          # doctest: +NORMALIZE_WHITESPACE
-    array(['U'], dtype='<U1')
-    >>> _to_ticks('..3')                        # doctest: +NORMALIZE_WHITESPACE
-    array([0, 1, 2, 3])
-    >>> _to_ticks('01..12')                     # doctest: +NORMALIZE_WHITESPACE
-    array(['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'], dtype='<U2')
-    >>> _to_ticks('01,02,03,10,11,12')          # doctest: +NORMALIZE_WHITESPACE
-    array(['01', '02', '03', '10', '11', '12'], dtype='<U2')
+    >>> list(_to_ticks('M , F'))                # doctest: +NORMALIZE_WHITESPACE
```

Comment:
do you know what broke those?

Reply:
With numpy 1.20, the dtype associated with the returned array by _to_ticks() differs according to the Python version and the machine! Sometimes it is '<U2', sometimes '<U21' or even 'U<11'.